include/multipass/platform.h

```diff
@@ -34,6 +34,7 @@ namespace multipass
 {
 namespace platform
 {
+QString preliminary_gui_autostart_setup(); // returns a platform-specific string
```

Comment:
Is this string only used for testing?

Reply:
Yes

Reply:
Testing will need directions of where to look for the autostart stuff in other platforms too, be it file or registry paths. Duplicating info in the test did not feel right, nor did creating a separate function or variable for that.

Reply:
Yeah I don't have a better approach right now…